TFT/src/User/API/boot.h

```diff
@@ -58,9 +58,9 @@ extern "C" {
 // This List is Auto-Generated. Please add new icons in icon_list.inc only
 enum
 {
-  #define X_ICON(NAME) ICON_##NAME ,
+  #define  X_ICON(NAME) ICON_##NAME ,
   #include "icon_list.inc"
-  #undef  X_ICON
```

Comment:
Please remove the double spaces after #define

Reply:
I aligned both #define, #include and #undef

Reply:
it does not make any sense to align them. #include is here is a sub-statement here.

Reply:
ok. I will revert them

Reply:
original #undef  X_ICON has double spaces. Shall I provide 1 spaces or what? It seems to me more coherent to align all or nothing

Reply:
There should be 1 space. It can be like this:
```C
#define X_ICON(NAME) ICON_##NAME ,
#include "icon_list.inc"
#undef X_ICON
```
or like this to show that icon_list.inc only relevant for this portion only
```C
#define X_ICON(NAME) ICON_##NAME ,
  #include "icon_list.inc"
#undef X_ICON
```

Reply:
ok for the second option (same convention used in all files)